API/Resources/UserProjectResourceResult.cs

```diff
@@ -0,0 +1,32 @@
+using Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Resources
+{
+    /// <summary>
+    /// Object to return to frontend with the UserProject
+    /// </summary>
+    public class UserProjectResourceResult
```

Comment:
I think more descriptive naming would be helpful. In this case, I think name of the class should've been **"UserProjectFollowResourceResult"** since the sole purpose of this class is return the data of projects that liked by users. Same for the service and the model class.
It would really help to understand the code for outsiders like me. Just an opinion.